packages/runtime.centos.7-x64.native.Microsoft.SqlToolsService/runtime.centos.7-x64.native.Microsoft.SqlToolsService.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Is this solution needed? Don't see it for any of the others

Reply:
The solution is indeed needed.  It determines the build order and dependencies of the nuget package.

Reply:
Why is there only this one then?

Reply:
The solution is used to coordinate the build of all of the projects.

Reply:
I expect that you can cakify it if you want.

Reply:
But it only links to one project...and the name is `runtime.centos.7-x64.native.Microsoft.SqlToolsService.sln`

Is there something I'm missing with how this is used that makes it apply to the other projects as well? If it is generic I guess I would have expected it to be in the root of the `packages` folder.

Reply:
oh, that solution shouldn't exist.  I sort of wonder how it got there. it can be deleted.